---
layout: cv
title: Curriculum Vitae
subtitle: Kai Zhang
---

:::::personal

## Personal data

:::::: {.grid-x #personal}

::: {.cell .medium-6}
* Department of Chemistry and Biochemistry
* University of California San Diego
* 9500 Gilman Drive
* San Diego, CA 92093, USA
:::

::: {.cell .medium-6 style="text-align: right"}
* email: kai@kzhang.org
* github: kaizhang
* website: https://kzhang.org
:::

::::::


### Research interests

I develop and apply systems biology approaches to study gene regulation. Specifically, I utilize genomics, epigenomics and single-cell genomics data to study transcriptional regulatory elements, construct gene regulatory networks and identify driver genes in different diseases and biological processes. 

:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::


:::education

## Education

2018
:   **Ph.D., Bioinformatics and Systems Biology, University of California San Diego, CA**

    - Thesis: Novel network-based integrated analyses of multi-omics data reveal new insights into CD8+ T cell differentiation and mouse embryogenesis
    - Advisor: Wei Wang

2012
:   **M.S., Biochemistry and Molecular Biology, Xiamen University, China**

    - Fields: Transcriptional elongation, signal transduction.

2009
:   **B.S., Biological Science, Xiamen University, China**


:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::

:::research-experience

## Research experience

2013 - 2018

:   Research Assistant, Bioinformatics and Systems Biology Program, University of California San Diego.

<!-- 
Developed an algorithm to identify protein complexes mediating
chromosome looping.
Developed an algorithm to perform the integrated analysis of
multi-omics data and identify driver transcription factors.
Explored innovative graph algorithms to address current systems
biology challenges.
Developed bioinformatics pipelines for analyzing omics data.
Participated an information retrieval challenge to develop better
machine learning models for finding the relevant information from
public databases.
-->

2012

:   Research Assistant, Department of Biochemistry, National University of Singapore, Singapore.

<!--
Performed bioinformatics analyses for next generation sequencing
data.
Studied the transcriptional regulation in stem cell differentiation.
-->

2008 - 2012

:   Research Assistant, Department of Biochemistry, Xiamen University, China.

<!--
Studied the molecular mechanisms of P-TEFb and Brd4 in
transcriptional elongation regulation.
Focused on the development of biochemical assays for analyzing
protein interactions with DNA.
Discovered the synergistic effect of HMBA and prostratin on
antagonizing HIV latency.
-->

:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::

:::publications

## Publications

A list is also available [online](https://scholar.google.com/citations?user=dV1fNBMAAAAJ&hl=en).

### Peer-Reviewed Articles

forthcoming

:   **Zhang, K.**, Wang, M., Zhao, Y., and Wang, W.
Systems-level identification of key transcription factors reveals transcriptional waves in mouse embryonic development.
▶*Science Advances*.

forthcoming

:   Ngo, V., Chen, Z., **Zhang, K.**, Whitaker J., Wang, M., Wang, W.
Epigenomic analysis reveals DNA motifs regulating histone modifications in human and mouse.
▶*PNAS*.

forthcoming

:   Fan, S., Tang, J., Li, N., Zhao, Y., Ai, R., **Zhang, K.**, Wang, M., Du, W.,
and Wang, W.
Integrative analysis with expanded DNA methylation data reveals common key regulators and pathways in cancers.
▶*Npj Genomic Medicine*.

2018

:   Wei, W., Ji, Z., He, Y., **Zhang, K.**, Ha, Y., Li, Q., and Ohno-Machado, L.
Finding relevant biomedical datasets: the UC San Diego solution for the bioCADDIE Retrieval Challenge.
▶*Database*. 

2018

:   Ainsworth, R., Ai, R., Ding, B., Li, N., **Zhang, K.**, and Wang, W.
Bayesian Networks Predict Neuronal Transdifferentiation.
▶*G3 Genes Genomes Genetics*.

2017

:   Milner, J., Toma, C., Yu, B., **Zhang, K.**, Omilusik, K., Phan, A., Wang,
D., Getzler, A., Nguyen, T., Crotty, S., et al.
Runx3 programs CD8+ T cell residency in non-lymphoid tissues and tumours.
▶*Nature*.

2017

:   Yu, B.\*, **Zhang, K.\***, Milner, J., Toma, C., Chen, R., Scott-Browne, J.,
Pereira, R., Crotty, S., Chang, J., Pipkin, M., et al.
Epigenetic landscapes reveal transcription factors that regulate CD8+ T cell
differentiation.
▶*Nature Immunology*. **(\* co-first authors)**

2016

:   **Zhang, K.**, Li, N., Ainsworth, R., and Wang, W.
Systematic identification of protein combinations mediating chromatin looping.
▶*Nature Communications*.

2016

:   Zhu, Y., Chen, Z., **Zhang, K.**, Wang, M., Medovoy, D., Whitaker, J.,
Ding, B., Li, N., Zheng, L., and Wang, W.
Constructing 3D interaction maps from 1D epigenomes.
▶*Nature Communications*.

2011

:   Chen, R., Liu, M., **Zhang, K.**, and Zhou, Q.
Isolation and functional characterization of P-TEFb-associated factors that control
general and HIV-1 transcriptional elongation.
▶*Methods*.

2011

:   Ai, N., Hu, X., Ding, F., Yu, B., Wang, H., Lu, X., **Zhang, K.**, Li, Y., Han, A., Lin, W., et al.
Signal-induced Brd4 release from
chromatin is essential for its role transition from chromatin targeting to
transcriptional regulation.
►*Nucleic Acids Res*.

:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::

:::honor

## Honors and awards

2015

: Doctoral fellowship, Frontiers of Innovation Scholars Program (FISP), UC San Diego.

2012-2014

: Doctoral fellowship, Dr. Huang Memorial Scholarship, UC San Diego.

:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::


:::professional-service

## Professional service

### Peer Review Activities

- Journal: Database, Journal of Integrative Bioinformatics, Mutation Research: Fundamental and Molecular Mechanisms of Mutagenesis.


:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::